src/main/kotlin/com/exactpro/th2/act/ssh/SshService.kt

```diff
@@ -42,6 +41,7 @@ import java.nio.file.Path
 import java.rmi.RemoteException
 import java.time.Duration
 import java.util.EnumSet
+import org.apache.sshd.common.util.io.output.NullOutputStream
```

Comment:
Why the import was reordered?

Reply:
My IDE put it this way. Package path changed also